src/tonic012.rs

```diff
@@ -1,6 +1,10 @@
+use tonic_012::transport::server::{Connected, TcpConnectInfo};
+
 #[cfg(all(feature = "unix", unix))]
 use tonic_012::transport::server::UdsConnectInfo;
-use tonic_012::transport::server::{Connected, TcpConnectInfo};
+
+#[cfg(all(any(target_os = "linux", target_os = "android", target_os = "macos"), feature = "vsock"))]
+use tokio_vsock::{VsockStream, VsockConnectInfo};
```

Comment:
(see note about duplicating the `VsockConnectInfo` here in the main discussion)

Reply:
Inlined that code as `vsock_support` module at end of file